src/helpers/mysql.ts

```diff
@@ -33,6 +34,8 @@ sequencerConfig.connectTimeout = 60e3;
 sequencerConfig.acquireTimeout = 60e3;
 sequencerConfig.timeout = 60e3;
 sequencerConfig.charset = 'utf8mb4';
+sequencerConfig.ssl = { rejectUnauthorized: true };
```

Comment:
```suggestion
```suggestion
if (process.env.NODE_ENV === 'production') {
  sequencerConfig.ssl = { rejectUnauthorized: true };
}
```
```